pkg/wfc3/calwf3/calwf3/wf3member.c

```diff
@@ -43,10 +42,9 @@ WF3Info *wf3		o: exposure specific flags and info
 
 	/* Initialize local strings */
 	rootname[0] = '\0';
-	outroot[0]  = '\0';
 	mtype[0]    = '\0';
 
-	strcpy (outroot, asn->product[prodid].subprod[posid].name);
+	const char * outroot = asn->product[prodid].subprod[posid].name;
```

Comment:
If you thought about it, then it's fine by me. But I would still want to wait and see if Megan has anything to add when she returns.

If you want the `SZ_LINE` stuff to go in 2017.2a now for testing, I would suggest you split these changes into 2 PRs.